Sources/TuistKit/Commands/GenerateCommand.swift

```diff
@@ -19,24 +19,52 @@ struct GenerateCommand: ParsableCommand, HasTrackableParameters {
     )
     var path: String?
 
+    @Argument(help: """
+    A list of targets in which you'd like to focus. \
+    Those and their dependant targets will be generated as sources. \
```

Comment:
```suggestion
    Other targets will be linked as binaries if possible. \
```

`generated as sources` sounds vague. I understand my suggested phrase is not exhaustive but if someone wants to dig deep, they should be able to find the answer in the documentation.